snap/README.md

```diff
@@ -114,35 +146,322 @@ Or by using the systemd unit name and `journalctl`:
 $ journalctl -u snap.edgexfoundry.consul
 ```
 
+### Configuring individual services
+
+All default configuration files are shipped with the snap inside `$SNAP/config`, however because `$SNAP` isn't writable, all of the config files are copied during snap installation (specifically during the install hook, see `snap/hooks/install` in this repository) to `$SNAP_DATA/config`.
+
+**Note** - `$SNAP` resolves to the path `/snap/edgexfoundry/current/` and `$SNAP_DATA` resolves to `/var/snap/edgexfoundry/current`.
+
+The preferred way to change the configuration is top use configuration overrides (see below). It is also possible to change
+configuration directly via Consul's UI or [kv REST API](https://www.consul.io/api/kv.html). Changes made to configuration in Consul
```

Comment:
nit: since Consul was locked down as part of Ireland, there's a bit more involved...

Reply:
yes - we can document that later. According to https://docs.edgexfoundry.org/2.0/security/Ch-Secure-Consul/ you need to use the bootstrap token, which could can be copied from /var/snap/edgexfoundry/current/secrets/consul-acl-token/bootstrap